NuGet.config

```diff
@@ -10,6 +10,8 @@
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-emsdk -->
     <add key="darc-pub-dotnet-emsdk-c7ba18b" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-c7ba18be/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-c7ba18b-2" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-c7ba18be-2/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-c7ba18b-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-c7ba18be-1/nuget/v3/index.json" />
```

Comment:
I'm not sure I understand why the xharness PRs are adding feeds for emsdk - @directhex @lewing do you know?

Reply:
This confuses me too, @mmitche what is going on here?